AppCenter/AppCenter/Internals/Ingestion/MSIngestionProtocol.h

```diff
@@ -26,6 +26,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly, getter=isReadyToSend) BOOL readyToSend;
 
+/**
+ * Send data.
+ *
+ * @param data Instance that will be transformed to request body.
+ * @param authToken Auth token to send data with.
+ * @param handler Completion handler.
+ */
+- (void)sendAsync:(nullable NSObject *)data
```

Comment:
This is going to be a break change in `feature/storage` branch. Please also consider to refactor classes inherited from this  class.

Reply:
Good point! We need to create a separate PR against `feature/storage` branch. For tracking we will create a separate user story to merge `feature/identity` to `feature/storage`, to resolve conflicts and breaking changes.